### ISSUE TITLE
Unmanaged Memory Leak

### ISSUE THREAD

**Issue by severmat52** [QuestPDF/QuestPDF#306]
**Describe the bug**
When creating large amounts of PDFs, the memory usage will continuously climb. Managed memory plateaus and is not an issue, but the unmanaged memory will continue increasing.

**To Reproduce**
I cloned the current main branch of this repository, and used the QuestPDF.ReportSample to show this. If you run this code and use a memory profiler, you should see the issue.

The way I am using this library, I need to grab the bytes instead of first writing to a file. I did test this by also writing to a file and got the same results.

```cs
using QuestPDF.Fluent;
using QuestPDF.ReportSample;
using QuestPDF.ReportSample.Layouts;

while (true)
{
    var model = DataSource.GetReport();
    var report = new StandardReport(model);
    var bytes = report.GeneratePdf();
}
```
**Expected behavior**
The unmanaged memory will continue increasing.

**Screenshots**
![image](https://user-images.githubusercontent.com/1002501/184440093-814c9f79-459f-43b3-bb73-a61c762f479b.png)

**Additional context**
I plan on using this library for an application that will need to produce large amounts of PDFs. I am actively researching this issue myself, but have not found the reason this is happening. But because this uses the SkiaSharp library that allocates unmanaged memory when rendering the PDFs, I have been focusing on this.

Any help is greatly appreciated. Thank you

**Comment by jnyrup** [comment]
Disposing `buffer` seems to stabilize the memory usage.
https://github.com/QuestPDF/QuestPDF/blob/db9d8b5fcfb357e1385eeb6d4725277ececd2881/QuestPDF/Drawing/TextShaper.cs#L24

I hunted this down manually, but [IDisposableAnalyzers](https://www.nuget.org/packages/IDisposableAnalyzers) also warns about `buffer` not being disposed.

**Comment by MarcinZiabek** [comment]
@severmat52 Thank you for finding this issue.
@jnyrup Thank you for finding the root cause. I wish there is an automated suggestion in Resharper for this type of issues...

I have fixed it in main, should be available in the 2022.8 release soon 😁

**Comment by severmat52** [comment]
Thanks @jnyrup and @MarcinZiabek, this is a very quick turnaround from when I reported the issue. Much appreciated!

**Comment by daniilda** [comment]
Gosh, that seems like the problem that have been bothering me for a few weeks. I was just about to report this issue and to start looking for the cause of it. Glad it is fixed

**Comment by MarcinZiabek** [comment]
This bug should be fixed in the 2022.8 release 😁
